lib/openers/evince-opener.js

```diff
@@ -0,0 +1,81 @@
+/** @babel */
+
+import Opener from '../opener'
+import dbus from 'dbus-native'
+import url from 'url'
+
+function SyncSource (uri, point) {
+  atom.workspace.open(url.parse(uri).pathname).then(editor => editor.setCursorBufferPosition(point))
+}
+
+export default class EvinceOpener extends Opener {
+  constructor () {
+    super()
+    this.bus = dbus.sessionBus()
+    this.daemon = null
+    this.windows = {}
+  }
+
+  getDaemon (callback) {
+    if (this.daemon) return callback(null, this.daemon)
+    this.bus.getInterface('org.gnome.evince.Daemon', '/org/gnome/evince/Daemon', 'org.gnome.evince.Daemon',
```

Comment:
I don't understand what these strings mean. Can we extract them as constants with meaningful names?